CHANGELOG.md

```diff
@@ -18,6 +20,11 @@ unaffected. (#684)
 `optionsTemplateURL` will display the template in the side navigation menu.
 This may require minor layout/appearance adjustments in the template. (#684)
 
++ To upgrade:
+  ++ If using the "app-title" attribute on frame-page directive, that title will be used as the document title
+  ++ If you want a heading on pages that formerly had the app-header, you'll need to use the "page-title" attribute on frame-page directive (or add your own <h1> tag to their pages wherever/however you want)
```

Comment:
I think this language we so often reach for about what "you want" is problematic. It's not about what you or I want. It's about building applications on behalf of the institution that serve users well. 

Do we really want to be encouraging app developers to sprinkle in ad-hoc `<h1>` tags when they could be feeding a `page-title` to `frame-page` and thereby making these titles available semantically in a way that might migrate forward in future releases?

So what about something more like

```
++ Pages formerly using `app-header` can use the `page-title` attribute on `frame-page` to achieve a similar within-page `h1` heading as before. 
```